PdfAValidator/PdfAValidator.cs

```diff
@@ -110,17 +111,24 @@ public async Task<Report> ValidateWithDetailedReportAsync(string pathToPdfFile)
         /// Validates a pdf and returns a detailed compliance report
         /// </summary>
         /// <param name="pathToPdfFile"></param>
-        /// <param name="otherArguments">Command line arguments</param>
+        /// <param name="commandLineArguments">Command line arguments</param>
         /// <returns></returns>
-        public async Task<Report> ValidateWithDetailedReportAsync(string pathToPdfFile, string otherArguments)
+        public async Task<Report> ValidateWithDetailedReportAsync(string pathToPdfFile, string commandLineArguments)
+        {
+            return await ValidateBatchWithDetailedReportAsync(commandLineArguments, pathToPdfFile).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Validates a batch of pdf files and returns a detailed compliance report
+        /// </summary>
+        /// <param name="commandLineArguments">Command line arguments</param>
+        /// <param name="pathsToPdfFiles"></param>
+        /// <returns></returns>
+        public async Task<Report> ValidateBatchWithDetailedReportAsync(string commandLineArguments, params string[] pathsToPdfFiles)
```

Comment:
Maybe i dont get it, but shouldnt "string[] pathsToPdfFiles" be prefered over "params string[] pathsToPdfFiles"?
What do you thing about aligning the order of the parameter to the single file call (pathsToPdfFiles first and commandLineArguments second)

I suggest:
public async Task<Report> ValidateBatchWithDetailedReportAsync(string[] pathsToPdfFiles, string commandLineArguments)

Reply:
Yes, that's very reasonable, I'll make those changes.

My idea was to reflect the actual call to veraPDF, but this fits better with the rest of the code.